src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Ball.Colors getRandomColor() {
+        return Ball.Colors.values()[new Random().nextInt(Ball.Colors.values().length)];
```

Comment:
We can add field with type `Random` to our class, so we won't need to create an instance every time when method invoked

Reply:
check previous comment, you didn't fix it